guestbook/comments/views.py

```diff
@@ -0,0 +1,90 @@
+from django.shortcuts import render
+from django.views.generic import ListView, DetailView
+from django.views.generic.edit import FormMixin
+from django.views.generic import CreateView, UpdateView, DeleteView
+from django.core.urlresolvers import reverse_lazy
+from django.contrib.auth.decorators import login_required
+from django.contrib.messages.views import SuccessMessageMixin
+from django.utils import timezone
+
+from .models import Comment, Message
+from .forms import AddMessageForm, CreateCommentForm
+
+
+class AllCommentList(SuccessMessageMixin, CreateView):
+    model = Comment
+    template_name = 'public_comments_list.html'
+    form_class = AddMessageForm
+    context_object_name = 'comments'
+    success_message = "Message was created successfully"
+
+    def form_valid(self, form):
+        comment_id = self.request.POST['comment_id']
+        form.instance.user = self.request.user
+        form.instance.comment = Comment.objects.get(pk=comment_id)
```

Comment:
Although this isn't particularly wrong I would move the `Comment.objects.get()` part into the `dispatch()` method.